Bottom: a0d7f878f73dfd0d4d6bc1521cf534a9356f9247
Top:    62dcc3c6fbf83d0457fbf3b263bd8a3268b7855e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:04:48 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/face3d.c b/src/hid/common/face3d.c
deleted file mode 100644
index e69de29..0000000
diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
deleted file mode 100644
index e69de29..0000000
